app/models/manageiq/providers/ibm_power_vc/inventory/parser/cloud_manager.rb

```diff
@@ -1,2 +1,210 @@
 class ManageIQ::Providers::IbmPowerVc::Inventory::Parser::CloudManager < ManageIQ::Providers::Openstack::Inventory::Parser::CloudManager
+  def parse_vm(vm, hosts)
+    super
+    server = persister.vms.find_or_build(vm.id.to_s)
+
+    # saving processor type and amount
+    flavor = get_flavor(vm)
+    processors = flavor.try(:vcpus)
+    persister.vms_and_templates_advanced_settings.build(
+      :resource     => server,
+      :display_name => _('Entitled Processors'),
```

Comment:
@kavyanekkalapu can you remind me how gettext works with purely backend code?  As far as I know `_("string")` will display the translated string on the UI (which won't do anything here since it is being saved to the database).

Is it `N_()` which marks the string for translation in the message catalog?

Reply:
Perfect thanks @kavyanekkalapu  so @Kuldip-Nanda can you change these to be e.g. `N_("Entitled Processors")`

Reply:
Thanks @kavyanekkalapu and @agrare. Yes sure will take care of this.